components/integration/staging/base/kustomization.yaml

```diff
@@ -11,6 +11,11 @@ images:
   newName: quay.io/redhat-appstudio/integration-service
   newTag: 55b2bae85419df8cd8b9e633d212826a7b9567ee
 
+configMapGenerator:
+- name: console-url
+  literals:
+    - CONSOLE_URL="https://console.dev.redhat.com/preview/preview/application-pipeline/ns/%s/pipelinerun/%s"
```

Comment:
Just noticed there are two previews in this link, should it be `preview/preview` -> `preview/`?

Reply:
Good catch!